flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -1005,6 +1013,11 @@ class TestingTableEnvironment private(
         } else {
           buffer(modifyOperations)
         }
+      case createOperation: CreateTableOperation =>
+        catalogManager.createTable(
+          createOperation.getCatalogTable,
```

Comment:
There is no need to extends `TableTestBase`, we should avoid to initialize the `TableEnvironment` as much as we can because it is an overhead.

Reply:
I agree with you. But we have to keep this `TestingTableEnvironment` for now, because we need the methods of `registerFunction` and `getBufferedOperations/clearBufferedOperations`. Please see the comment above that methods.